week2homework.js

```diff
@@ -0,0 +1,121 @@
+//1
+let EnglishGreeting = "Hello world!";
+console.log (EnglishGreeting)
+let ArabicGreeting = "مرحباً بالعالم";
+console.log (ArabicGreeting)
+let ItalianGreeting = "Ciao Mondo!";
+console.log (ItalianGreeting)
+let TurkishGreeting = "Selam Dünya!";
+console.log (TurkishGreeting)
+let DutchGreeting = "Hallo Wereld!";
+console.log (DutchGreeting)
+
+//2
+console.log ('I\'m awesome');
+
+//3
+let x ;
+console.log ("the value of my variable x will be:EvenNumber");
+console.log (x);
+x = 8 ;
+console.log ("the value of my variable x will be:EvenNumber");
+console.log (x)
+
+//4
+let y = "HackYourFuture"
+console.log ("the value of the string is "+"HackYourFuture")
+console.log (y)
+y = "HYF"
+console.log ("the value of the string is "+"HYF")
+console.log (y)
+
+//5
+let z = 7.25;
+console.log(z);
+let a = (Math.round(z));
+console.log(a);
+
+let highest;
+if (z>a){
+    highest=z;
+}
+else{
+    highest=a;
+}
+console.log(highest);
+
+
+//6
+let FavoriteAnimals = [ ];   //6.1
+console.log("the value of my array is : Empty."); //6.2
+console.log(FavoriteAnimals); //6.3
+let MyFavoriteAnimals = [ 'Deer' , 'Horse' , 'Koala' ]; //6.4
+console.log (MyFavoriteAnimals);  //6.5
+let NewMyFavoriteAnimals = MyFavoriteAnimals.concat(['baby pig']); //6.6
```

Comment:
Ah nice, you can also use 
`MyFavoriteAnimals.push('baby pig');`

Which might be an easier solution